.travis.yml

```diff
@@ -19,8 +19,11 @@ before_script:
   - version="$(awk '$2 == "FREESWITCH_VERSION" { print $3; exit }' Dockerfile | cut -d'~' -f1)"
   - docker pull "$IMAGE"
 
-# Build the container, run it, wait for it to start, then stop it.
+# Build the modules container runs it, modules are copied to build/modules.
+# Build the freeswitch container, run it, wait for it to start, then stop it.
 script:
+  - docker build -f Dockerfile.modules -t fs-modules .
```

Comment:
I've been naming Dockerfiles like `modules.dockerfile` which more things seem to pick up and do the right syntax highlighting for (including Github).